dlt/destinations/dataset/ibis_relation.py

```diff
@@ -85,6 +88,24 @@ def columns_schema(self) -> TTableSchemaColumns:
     def columns_schema(self, new_value: TTableSchemaColumns) -> None:
         raise NotImplementedError("columns schema in ReadableDBAPIRelation can only be computed")
 
+    @property
+    def table_name(self) -> str:
+        return self._table_name
+    
+    @cached_property
+    def load_table(self) -> "ReadableIbisRelation":
```

Comment:
You should be able to get the loads table with self["_dlt_loads"] You don't need to construct any tables yourself, this class should be able to do all of this for you.